kotlin/PixelPass/build.gradle.kts

```diff
@@ -33,6 +33,12 @@ android {
 }
 
 dependencies {
+    implementation(files("libs/cose-java-1.1.0.jar"))
```

Comment:
We don't have this from maven is it?

Reply:
The published library version has some issues. Dhanendra shared the jar file he had built from their repository locally